src/current/_includes/common/network-partitions.md

```diff
@@ -0,0 +1,7 @@
+A network partition occurs when two or more nodes are prevented from communicating with each other in one or both directions. A network partition can be caused by a network outage or a configuration problem with the network, such as when allowlisted IP addresses or hostnames change after a node is torn down and rebuilt.
```

Comment:
Do we have a more precise term than "torn down" at this point? Does it mean "drained and restarted"?

Reply:
SGTM, updated as suggested - thanks!